lib/note-list/index.tsx

```diff
@@ -91,7 +91,7 @@ const renderNote = (
     ) : (
       <CellMeasurer
         cache={heightCache}
-        columnIndex={-3}
+        columnIndex={0}
```

Comment:
my only suggestion here is that we make sure this doesn't interfere with the tag suggestions which also uses `columnIndex={0}`. when reading on `CellMeasurer` I had a hard time truly understanding the role in play here; there's some association with the caches we clear in `deriveStateFromProps`

Reply:
As per Slack discussion, the -3 index was an attempt to set it out of bounds so it wouldn't conflict with the notes in the list, but doesn't seem to work that way and causes other issues. Since this tests out okay and the height cache glitch isn't new, gonna merge this and we can keep arguing with CellMeasurer another day.